smart_card_connector_app/src/window-main.js

```diff
@@ -24,6 +24,7 @@ goog.require('GoogleSmartCard.ConnectorApp.Window.AboutShowing');
 goog.require('GoogleSmartCard.ConnectorApp.Window.AppsDisplaying');
 goog.require('GoogleSmartCard.ConnectorApp.Window.DevicesDisplaying');
 goog.require('GoogleSmartCard.ConnectorApp.Window.LogsExporting');
+goog.require('GoogleSmartCard.ConnectorApp.Window.HelpShowing');
```

Comment:
nit: Move this to be between "DevicesDisplaying" and "LogsExporting", to keep the list sorted.

(BTW, hmm, I thought that we have a diagnostics enabled in the Closure Compiler that throws an error when the order is violated. Maybe it stopped working though.)